public/css/main.css

```diff
@@ -1,20 +1,357 @@
 @font-face {
   font-family: 'squealerregular';
   src: url('fonts/squealer-webfont.eot');
-  src: url('fonts/squealer-webfont.eot?#iefix') format('embedded-opentype'),
-    url('fonts/squealer-webfont.woff2') format('woff2'),
-    url('fonts/squealer-webfont.woff') format('woff'),
-    url('fonts/squealer-webfont.ttf') format('truetype'),
-    url('fonts/squealer-webfont.svg#squealerregular') format('svg');
+  src: url('fonts/squealer-webfont.eot?#iefix') format('embedded-opentype'), url('fonts/squealer-webfont.woff2') format('woff2'), url('fonts/squealer-webfont.woff') format('woff'), url('fonts/squealer-webfont.ttf') format('truetype'), url('fonts/squealer-webfont.svg#squealerregular') format('svg');
   font-weight: normal;
   font-style: normal;
 }
 
 body {
   background-image: radial-gradient(circle, #3241a6 0, #202a6b 120%);
-  font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI",
-    "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue",
-    "Arial", sans-serif;
+  font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue", "Arial", sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+
+/*Cover removed as it required the background to be black*/
+
+
+/*Style the cover*/
+
+
+/*.cover {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.1);
+  display: block;
+  z-index: -1;
+  background: transparent url(../img/cover.png) repeat top center;
+  animation: moveCover 75s linear infinite;
+  -moz-animation: moveCover 75s linear infinite;
+  -ms-animation: moveCover 75s linear infinite;
+  -webkit-animation: moveCover 75s linear infinite;
+}*/
+
+
+/*Move Cover*/
+
+
+/*@keyframes moveCover {
+  from {
+    background-position: 0 0;
+  }
+  to {
+    background-position: -10000px 5000px;
+  }
+}
+
+@-ms-keyframes moveCover {
+  from {
+    background-position: 0 0;
+  }
+  to {
+    background-position: -10000px 5000px;
+  }
+}
+
+@-moz-keyframes moveCover {
+  from {
+    background-position: 0 0;
+  }
+  to {
+    background-position: -10000px 5000px;
+  }
+}
+
+@-webkit-keyframes moveCover {
+  from {
+    background-position: 0 0;
+  }
+  to {
+    background-position: -10000px 5000px;
+  }
+}*/
+
+
+/*Style stars*/
+
+.stars {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  width: 100%;
+  /*background: rgba(0, 0, 0, 0.1);*/
+  display: block;
+  background: transparent url(../img/stars.png) repeat top center;
+  z-index: -1;
+  animation: moveStars 20s linear infinite;
+  -ms-animation: moveStars 20s linear infinite;
+  -moz-animation: moveStars 20s linear infinite;
+  -webkit-animation: moveStars 20s linear infinite;
+}
```

Comment:
I think the majority of this can be done using utility classes, are you able to do that? See http://tachyons.io/docs/ for info. I imagine the animation and background you'll need to keep here but the rest I think can be done using utility classes.